pele/mindist/periodic_mindist.py

```diff
@@ -64,13 +57,18 @@ def __call__(self, coords1, coords2):
 
     def finalize_best_match(self, x1, best_x2):
         ''' do final processing of the best match '''
-        ca = CoordsAdapter(coords=best_x2)
-        ca.posRigid -= np.round(ca.posRigid / self.boxvec) * self.boxvec
-
-        dist = self.measure.get_dist(x1, best_x2)
+        x1 = x1.reshape(-1,3)
+        dx = x1 - best_x2
+        dx = np.round(dx / self.boxvec) * self.boxvec
```

Comment:
For the sake of generality we should think about replacing this with a call to a new function `measure.get_distance_vector()`.